utils/api.electron.js

```diff
@@ -0,0 +1,19 @@
+import electron from 'electron'
+
+let electronStore = electron.remote.getGlobal('sharedObj').electronStore
```

Comment:
Does require return the same instance of store in the main process?

Reply:
yes. basically I am using the instance I created with new in the main process and assigned to this sharedObj dict.

Reply:
OK, it would be better to put electronStore directly in global instead of sharedObj. As getGlobal does sync operation and this affects performance, so this will result in copying the whole object through IPC and we will only use electronStore.